module/proto/gravity/v1/query.proto

```diff
@@ -87,6 +87,9 @@ service Query {
   rpc GetAttestations(QueryAttestationsRequest) returns (QueryAttestationsResponse) {
     option (google.api.http).get = "/gravity/v1beta/query_attestations";
   }
+  rpc GetERC721Attestations(QueryERC721AttestationsRequest) returns (QueryERC721AttestationsResponse) {
+    option (google.api.http).get = "/gravity/v1beta/query_attestations";
```

Comment:
Shouldn't we have two different paths for different attestiation queries? Or maybe a one query with additional param to distinguish between old Gravity events and ERC721 events?

Reply:
Ups yes.

I was considering reusing the same query, but it was just faster to do it this way now. Potential for change later. :)